web/modules/custom/ct_manager/src/Plugin/QueueWorker/ProcessUsers.php

```diff
@@ -90,13 +90,15 @@ public function processItem($data) {
     /** @var \Drupal\ct_manager\ContributionSourceInterface $plugin_instance */
     $plugin_instance = $this->pluginManager->createInstance($plugin_id);
     if (!($plugin_instance->isUserValid($user))) {
-      $this->logger->error('@plugin username for @username is invalid.', ['@plugin' => $plugin_id, '@username' => $user->getAccountName()]);
+      $this->logger->notice('@plugin username for @username is invalid.', ['@plugin' => $plugin_id, '@username' => $user->getAccountName()]);
       return;
     }
 
     $issues = $plugin_instance->getUserIssues($user);
     $comments = $plugin_instance->getUserCodeContributions($user);
-
+    if (!$issues) {
+      return;
+    }
```

Comment:
Why is this needed? IIRC, `$issues` will always be an array and the `foreach` won't do anything. I am avoiding this `return` because there might be more things to run later. Are there cases where not having this throws an error?